src/Game/AccountManagement.java

```diff
@@ -271,20 +530,422 @@ public static void panel_welcome_hook(int n) {
+	        n3 += 35;
+	        Panel.addButtonBackTo(Client.panelContactDetails,n, n3, n2, 30);
+	        Panel.addCenterTextTo(
+                    Client.panelContactDetails,n, n3 - 7, "Country", 1, true);
+	        Client.countryInput = Panel.addInputTo(Client.panelContactDetails,n, n3 + 7, n2, 30, 1, 80, false, true);
+	        n3 += 35;
+	        Panel.addButtonBackTo(Client.panelContactDetails,n, n3, n2, 30);
+	        Panel.addCenterTextTo(
+                    Client.panelContactDetails,n, n3 - 7, "Email address", 1, true);
+	        Client.emailInput = Panel.addInputTo(Client.panelContactDetails,n, n3 + 7, n2, 30, 1, 80, false, true);
+	        n3 += 35;
+	        Panel.addButtonBackTo(Client.panelContactDetails,n, n3, 100, 30);
+	        Panel.addCenterTextTo(
+                    Client.panelContactDetails,n, n3, "Submit", 4, true);
+	        Client.chooseSubmitContactDetailsButton = Panel.addButtonTo(Client.panelContactDetails,n, n3, 100, 30);
+	        Panel.setFocus(Client.panelContactDetails,Client.fullNameInput);
+	  } catch (Exception e) {
+		  e.printStackTrace();
+	  }
+  }
+
+  public static int options_security_hook(int xPos, int yPos, int mouseX, int mouseY) {
+    short uiWidth = 196;
+    int currYPos = yPos;
+    if (Settings.SHOW_ACCOUNT_SECURITY_SETTINGS.get(Settings.currentProfile)) {
+      currYPos += 5;
+      Renderer.drawString("Security settings", xPos, currYPos, 1, 0);
+      currYPos += 15;
+      int textColor = 16777215;
+      if (mouseX > xPos
+          && mouseX < xPos + uiWidth
+          && mouseY > currYPos - 12
+          && mouseY < currYPos + 4) {
+        textColor = 16776960;
+      }
+      Renderer.drawString("Change password", xPos, currYPos, 1, textColor);
+      currYPos += 15;
+      textColor = 16777215;
+      if (mouseX > xPos
+          && mouseX < xPos + uiWidth
+          && mouseY > currYPos - 12
+          && mouseY < currYPos + 4) {
+        textColor = 16776960;
+      }
+      Renderer.drawString("Change recovery questions", xPos, currYPos, 1, textColor);
+      currYPos += 15;
+      textColor = 16777215;
+      if (mouseX > xPos
+          && mouseX < xPos + uiWidth
+          && mouseY > currYPos - 12
+          && mouseY < currYPos + 4) {
+        textColor = 16776960;
+      }
+      Renderer.drawString("Change contact details", xPos, currYPos, 1, textColor);
+      currYPos += 15;
+      // currYPos += 5; //originally had this extra space but makes moves privacy settings and
+      // subsequent block
+      // a bit down with respect to when SHOW_ACCOUNT_SECURITY_SETTINGS is not enabled
+    }
+    return currYPos - yPos;
+  }
+
+  public static int options_security_click_hook(
+      int xPos, int yPos, int mouseX, int mouseY, int mouseButtonClick) {
+    short uiWidth = 196;
+    int currYPos = yPos;
+    if (Settings.SHOW_ACCOUNT_SECURITY_SETTINGS.get(Settings.currentProfile)) {
+      currYPos += 15;
+      currYPos += 20;
+      if (mouseX > xPos

```

Comment:
Add comment that this is `Security settings -> Change password` found in mudclient175

Reply:
fixed